application/build.gradle

```diff
@@ -13,6 +13,7 @@ dependencies {
     api platform(project(':corda-api'))
     api project(":base")
     api project(":crypto")
+    api project(":data:avro-schema")
```

Comment:
This is a red flag sorry!

Reply:
That's exactly why I'm trying to move the whole API into a separate module as @relyafi suggested above. The whole conversation originated from that question on Slack.

Reply:
https://github.com/corda/corda-api/pull/518#discussion_r955164226